fw/ss_skb.h

```diff
@@ -256,6 +256,19 @@ ss_skb_alloc(size_t n)
 	return skb;
 }
 
+/* Move @cnt existing fragment descriptors within the SKB.
+ */
+static inline void
+ss_skb_frags_move(struct sk_buff *skb, unsigned int new_pos,
```

Comment:
I implement such function in PR 1845, but for moving frags from one skb to another. May be think about one version which is good for both this cases?

Reply:
I would leave it as it is for now. You can extend it as part of #1845 later.